hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapReorderedReplicationTest.java

```diff
@@ -193,4 +189,16 @@ public void run() throws Exception {
             super.run();
         }
     }
+
+    private void updateFactoryField(DataSerializableFactory factory) throws Exception {
+        // Get Unsafe object
+        final Field unsafeField = Unsafe.class.getDeclaredField("theUnsafe");
+        unsafeField.setAccessible(true);
+        final Unsafe unsafe = (Unsafe) unsafeField.get(null);
+
+        // Update the field
+        final Object staticFieldBase = unsafe.staticFieldBase(field);
+        final long staticFieldOffset = unsafe.staticFieldOffset(field);
+        unsafe.putObject(staticFieldBase, staticFieldOffset, factory);
```

Comment:
```suggestion
        // Update the field
        final Object staticFieldBase = UnsafeUtil.UNSAFE.staticFieldBase(field);
        final long staticFieldOffset = UnsafeUtil.UNSAFE.staticFieldOffset(field);
        UnsafeUtil.UNSAFE.putObject(staticFieldBase, staticFieldOffset, factory);
```
we can use unsafe util here. Also, could you handle the imports?